### ISSUE TITLE
Pre-commit doesn't pass

### ISSUE THREAD

**Issue by mhauru** [alan-turing-institute/python-project-template#14]
After running the template creation, some of the resulting files don't pass the pre-commit configuration created by the template.

**Comment by phinate** [comment]
Right, I feared this would happen at some point. I think the fix is to implement #4.

**Comment by phinate** [comment]
Still ongoing — small formatting changes currently preventing this. Would be awesome if there was a way to auto-push any corrections to the files pre-generation, but that feels like a _lot_ of meta-handling...
